include/rtpmidid/rtpclient.hpp

```diff
@@ -59,12 +60,14 @@ class rtpclient {
   /// every 10 secs.
   uint8_t timerstate;
 
+  std::optional<int> conn_event;
```

Comment:
Sounds good, but maybe just ensuring conn_event = 0 is an invalid state at the signal.hpp (start giving ids a1, disconnect from 0 does nothing) would be enough, and we avoid this extra optional.